src/main/native/windows/file.cc

```diff
@@ -467,6 +463,13 @@ int CreateSymlink(const wstring& symlink_name, const wstring& symlink_target,
 
   if (!CreateSymbolicLinkW(name.c_str(), target.c_str(),
                            symlinkPrivilegeFlag)) {
+    if (GetLastError() == ERROR_INVALID_PARAMETER) {
```

Comment:
Nit: This means on older Windows versions with developer mode enabled, each symlink operation triggers CreateSymlinkLink twice, which has some performance implication. But I guess it's small issue which won't impact the vast majority of Windows users.